rerun_py/docs/gen_common_index.py

```diff
@@ -476,6 +474,22 @@ def make_slug(s: str) -> str:
 Checkout [SDK Operating Modes](https://www.rerun.io/docs/reference/sdk/operating-modes) for an
 overview of what's possible and how.
 
+## Supported Python Versions
+
+Rerun will typically support Python version up until their end-of-life. If you are using an older version
+of Python, you can use the table below to make sure you choose the proper Rerun version for your Python installation.
+
+| **Rerun Version** | **Release Date** | **Supported Python Version** |
+|-------------------|------------------|------------------------------|
+| 0.25              | Sep./Oct. 2025   | 3.10+                        |
```

Comment:
we're still going to support 3.9, but will mark it as deprecated. Guess that's still appropriate to call it out as the not supported version since we'll print a warning 🤔